### ISSUE TITLE
Can you upgrade the bson version to 0.4.1?

### ISSUE THREAD

**Issue by pyros2097** [mongodb-labs/mongo-rust-driver-prototype#190]
I'm getting this error,
```
Compiling rstt v0.1.0 (file:///home/pyros2097/Code/rstt)
error[E0308]: mismatched types
  --> src/models.rs:18:23
   |
18 |       coll.insert_one(self.doc().unwrap(), None).unwrap()
   |                       ^^^^^^^^^^^^^^^^^^^ expected struct `bson::ordered::OrderedDocument`, found a different struct `bson::ordered::OrderedDocument`
   |
   = note: expected type `bson::ordered::OrderedDocument` (struct `bson::ordered::OrderedDocument`)
              found type `bson::ordered::OrderedDocument` (struct `bson::ordered::OrderedDocument`)
note: Perhaps two different versions of crate `bson` are being used?
  --> src/models.rs:18:23
   |
18 |       coll.insert_one(self.doc().unwrap(), None).unwrap()
   |                       ^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `rstt`.
```

Or atleast re-export bson module?

**Comment by saghm** [comment]
@kyeah, you were going to handle this as part of #186, correct?

**Comment by kyeah** [comment]
hey! I can upgrade the bson library without updating rustc_serialize stuff on the mongo end for now. Will do a release once tests pass.

**Comment by kyeah** [comment]
released v0.2.2. thanks for initiating, @pyros2097 ! 🙇
